lib/quest.coffee

```diff
@@ -39,6 +40,13 @@ should_redirect = (options, resp) ->
   options.method not in ['PATCH', 'PUT', 'POST', 'DELETE']))
 
 quest = (options, cb) ->
+  if cb == undefined # making this explicit due to problems with coffeescript 1.9.3
```

Comment:
After switching mocha to using coffeescript 1.9.3, I can't seem to reproduce the problem. Changing to normal coffeescript syntax.